docker/config/webapp.env

```diff
@@ -3,6 +3,7 @@ DJANGO_DEV=True
 DJANGO_DEBUG=True
 DATABASE_URL=postgres://pontoon:asdf@postgresql/pontoon
 SESSION_COOKIE_SECURE=False
+CSRF_COOKIE_SECURE=False
```

Comment:
Why do we need to explicitly disable this in DEV environment, given that it was enabled until now?